BinarySearch.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+Time complexity: O(log n)
+Contributor: Deepanshu Jindal(https://github.com/ultimatecoder2) 
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int BinarySearch(int *arr,int low,int high,int el)
+{
+    if(low>high)
+    return -1;
+    int mid  = (low+high)/2;
+    if(arr[mid]==el)
+        return mid;
+    else if(arr[mid]<el)
+        return BinarySearch(arr,mid+1,high,el);
+    else if(arr[mid]<el)
```

Comment:
check this condition once.

Reply:
Thanks for telling. I have updated the code. There should be > sign in the last else if condition..

Reply:
Kindly check once again...